src/database/mod.rs

```diff
@@ -12,21 +12,40 @@ pub use transaction::*;
 
 use crate::DbErr;
 
+#[derive(Debug)]
+pub enum DbScheme {
+    Postgres,
+    Mysql,
+    Sqlite,
+}
```

Comment:
Thanks!! @baoyachi 

- Perhaps we can add the `starts_with()` method to `DatabaseBackend` instead of introducing a new enum `DbScheme`?
- And rename the `starts_with()` method to `url_starts_with()`?

https://github.com/SeaQL/sea-orm/blob/1eb0e5cbb278413b327b1512e66b591ef66cffb7/src/database/connection.rs#L19-L24

Reply:
It's good 👍 .